ui/src/components/view/SettingsTab.vue

```diff
@@ -96,7 +96,7 @@ export default {
       filter: ''
     }
   },
-  beforeMount () {
+  created () {
```

Comment:
@utchoang @shwstppr @davidjumani - I similar change in quite a few recent PRs - should we check/scan/fix all components?

Reply:
Ping ^^ @utchoang @shwstppr @davidjumani @Pearl1594

Reply:
That might require considerable time testing. IMO, fix it as it comes along @rhtyd